hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,25 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB mass storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+
+<a name="pi3plus"></a>
+## Raspberry Pi 3B+, CM3+
+
+The Raspberry Pi 3B+ and CM3+ support USB mass storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is the same [procedure](../../../installation/installing-images) as for SD cards.
+
+After preparing the storage device, connect the drive to the Raspberry Pi and power up the Pi (be aware of the extra USB power usage from the external drive).
+After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
 
-Copy a Raspbian or Raspbian Lite installation image directly onto your USB device, in the same way that you would for an SD card. To perform this step, follow the instructions [here](../../../installation/installing-images/README.md), remembering to select the drive that corresponds to your USB mass storage device. Make sure that you use a recent version of Raspbian/Raspbian Lite.
+<a name="pi4"></a>
+## Raspberry Pi 4
 
-Once you have finished imaging your USB mass storage device, remove it from your computer and insert it into your Raspberry Pi.
+The Raspberry Pi 4's bootcode is stored in [EEPROM](../booteeprom.md) and can be updated. Support for mass storage boot will be added in a future update.
 
-## Boot your Raspberry Pi from the USB mass storage device
+## Known issues
 
-Attach the USB mass storage device to your Raspberry Pi, and power up the Pi. After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
```

Comment:
Please add this paragraph back in somewhere.

Reply:
It's been moved up to earlier in the page?

Reply:
My mistake - I saw it in red and assumed it had gone.